dbt/adapters/databricks/connections.py

```diff
@@ -58,6 +61,27 @@
 
 logger = AdapterLogger("Databricks")
 
+
+class DbtCoreHandler(logging.Handler):
+    def __init__(self, level: Union[str, int], *args: Tuple, **kwargs: Mapping):
+        super().__init__(level=level)
+        self.logger = AdapterLogger("databricks-sql-connector")
+
+    def emit(self, record: logging.LogRecord) -> None:
```

Comment:
No tests?

Reply:
Ugh, testing logs is a pain in the neck. Especially when they're configured by environment variables. I'll make a follow-up to this.

Reply:
See my last comment, I don't consider a blocker